mod_reforged/hooks/entity/tactical/humans/mercenary_ranged.nut

```diff
@@ -21,17 +21,14 @@
 		this.m.Skills.add(this.new("scripts/skills/perks/perk_coup_de_grace"));
 		this.m.Skills.add(this.new("scripts/skills/perks/perk_fast_adaption"));
 		this.m.Skills.add(this.new("scripts/skills/perks/perk_nimble"));
-		// this.m.Skills.add(this.new("scripts/skills/perks/perk_overwhelm"));
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_overwhelm"));
 		// this.m.Skills.add(this.new("scripts/skills/actives/rotation")); // Replaced with perk
 		// this.m.Skills.add(this.new("scripts/skills/actives/footwork")); // Replaced with perk
 		// this.m.Skills.add(this.new("scripts/skills/actives/recover_skill"));	// Now granted to all humans by default
 
 		// Reforged
-		b.RangedDefense += 10;
-		this.m.Skills.add(::new("scripts/skills/perks/perk_mastery_dagger"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_rotation"));
 		this.m.Skills.add(::new("scripts/skills/perks/perk_footwork"));
-		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_through_the_ranks"));
```

Comment:
Through the Ranks should be maintained on enemies, so that when their shots go astray, they are less likely to hit their allies. This is because enemies are far more reckless in terms of shooting through allies. This also helps the player when ally mercenaries shoot through players - otherwise players get hit by allied mercenary crossbows.

Reply:
resolved by giving through the ranks back